docs/modules/ROOT/pages/vshn-managed/minio.adoc

```diff
@@ -0,0 +1,10 @@
+= Minio by VSHN
+
+APPUiO Managed clusters will use the underlying cloud's Object Storage to provide Object Storage by default.
```

Comment:
There's too much confusion for my taste in this end user docs. I am myself not sure what exactly we should expect here in this page other than saying it's not self-service and the customer should contact us. I believe all this ObjectStorage solution from various providers should not be described here. Since @tobru requested this page he might have more input here.